docs/io-use.md

```diff
@@ -466,6 +466,14 @@ For the latest and complete information, see [Pulsar admin docs](pathname:///ref
 </Tabs>
 ````
 
+## Run a Pulsar Function before a sink connector
```

Comment:
@cbornet I don't quite understand this user task from the heading. 
Can you pls also attach the feature PR in this PR's description?

Reply:
Done. PTAL